tests/shouldwork/Cores/Xilinx/Floating.hs

```diff
@@ -123,13 +102,13 @@ addBasic
   :: Clock XilinxSystem
   -> DSignal XilinxSystem 0 Float
   -> DSignal XilinxSystem 0 Float
-  -> DSignal XilinxSystem F.AddDefDelay Float
-addBasic clk x y = withClock clk $ withEnable enableGen $ F.add x y
+  -> DSignal XilinxSystem F.AddDefDelay FloatVerifier
+addBasic clk x y = withClock clk $ withEnable enableGen $ FloatVerifier <$> F.add x y
```

Comment:
Hmmm, stylistically I think `FloatVerifier` is part of the test bench, not of the DUT.